### PR TITLE
Updated lists 05th of January

### PR DESCRIPTION
**Pull request by emmamarichal**
Lists updated - cc @m4rc1e if you want to check

### CODE REVIEW

**Comment by emmamarichal** [comment]
I'm not sure about the #deleted lines in the to_sandobox.txt, should I remove them?

**Review by m4rc1e — APPROVED** [review]
LGTM

**Comment by emmamarichal** [comment]
@chrissimpkins do you want to take a look before we merge?

File: to_production.txt [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
 
 # Designer profile
+catalog/designers/denismasharov # https://github.com/google/fonts/pull/6992
+catalog/designers/jenskutilek # https://github.com/google/fonts/pull/6935
+catalog/designers/vivarado # https://github.com/google/fonts/pull/7067
 
 # Axis Registry
+axisregistry/bleed.textproto # https://github.com/google/fonts/pull/6953
+axisregistry/scanlines.textproto # https://github.com/google/fonts/pull/6953
+axisregistry/shadow_length.textproto # https://github.com/google/fonts/pull/6953
```
Comment (chrissimpkins) [inline to_production.txt:25]: This is necessary for Honk.

File: to_production.txt [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+ofl/ysabeauoffice # https://github.com/google/fonts/pull/7036
+ofl/ysabeausc # https://github.com/google/fonts/pull/7037
 
 # Designer profile
+catalog/designers/denismasharov # https://github.com/google/fonts/pull/6992
+catalog/designers/jenskutilek # https://github.com/google/fonts/pull/6935
+catalog/designers/vivarado # https://github.com/google/fonts/pull/7067
 
 # Axis Registry
+axisregistry/bleed.textproto # https://github.com/google/fonts/pull/6953
```
Comment (chrissimpkins) [inline to_production.txt:23]: Bleed and scanlines are necessary for Sixtyfour & Workbench families

File: to_production.txt [hunk 3]
```diff
@@ -1,11 +1,35 @@
 # New
```
Comment (chrissimpkins) [inline to_production.txt:1]: I tested all of these families in Docs. Gtg

**Comment by chrissimpkins** [comment]
@emmamarichal Is Honk ready to move to production?  We are moving one of the custom axes required to support it here.

**Comment by emmamarichal** [comment]
Yes, I unblocked it, it works well now!

**Pull request merged**